Podfile

```diff
@@ -9,6 +9,7 @@ end
 
 target "ios-sdk" do
   shared_pods
+  pod "HockeySDK", :subspecs => ['CrashOnlyLib']
```

Comment:
Let's lock this to a specific version series like `pod "HockeySDK", ~> "4.1", :subspecs => ['CrashOnlyLib']` 

Would rather not get surprised by a major version release update.